### DISCUSSION TITLE
Map rotation 45°, 135°, 225°, 315° or custom value

### DISCUSSION THREAD

**Discussion by adrgumula** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#376] [Ideas]
After extracting the map it looks like on the left img I longing for a  feature to rotate the map 45 deg (or custom value) and then crop it.
It will allow me to set up rooms/zones in the natural way (on the rotated one) - right img.

![208298433-6b18a6a8-d84b-46b7-9b79-00ab8c7c529e](https://user-images.githubusercontent.com/70687019/208309541-4ca9fae9-766c-48b8-8ac4-9c3e8626f48c.png)

Thx,
Adrian

**Comment by PiotrMachowski** [comment]
Please create a feature request here: https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor/issues/new/choose
